### ISSUE TITLE
Will automatically trigger the handler non-stop！！！

### ISSUE THREAD

**Issue by chenpaner** [heisenshark/blender-krita-link-plugin#17]
There is a handler in my own plugin, but this plugin will cause it to fire constantly, but I am not connected to krita! I see a lot of bpy.app timers in the plugin, I don't understand why they trigger my handler! Can this plugin add a master switch in blender, if I turn off the master switch it has no background operation！
This is my  plugin`s code,it`s will  print("_update_cutoff_distance")
```
@persistent
def depsgraph_update_handler_update_cutoff_distance(self, context):
    pref=bpy.context.preferences.addons[__package__].preferences
    if bpy.context.scene.render.engine == 'CYCLES' and pref.use_cutoff_distance==True:
        cutoff_distance = pref.cutoff_distance
        light_gizmo_active_cutoff_distance = pref.light_gizmo_active_cutoff_distance
        activelight = bpy.context.view_layer.objects.active if bpy.context.view_layer.objects.active.type == 'LIGHT' else None
        try:
            for obj in bpy.context.scene.objects:
                if obj.type == 'LIGHT' and obj != bpy.context.active_object and hasattr(obj.data, 'cutoff_distance'):
                    obj.data.cutoff_distance = cutoff_distance
            if activelight and hasattr(activelight.data, 'cutoff_distance'):
                if activelight.select_get():
                    activelight.data.cutoff_distance = light_gizmo_active_cutoff_distance
                else:
                    activelight.data.cutoff_distance = cutoff_distance
            print("_update_cutoff_distance")
        except Exception as err:
            print('-----', err, end=' | ')
            print('ID-Tools Error on line {}'.format(sys.exc_info()[-1].tb_lineno))

            
bpy.app.handlers.depsgraph_update_post.append(depsgraph_update_handler_update_cutoff_distance)
```

https://github.com/user-attachments/assets/32982e2b-fb85-4466-87f0-23edcf6d41f0

**Comment by heisenshark** [comment]
your attachment does not work, also Is it breaking stuff in blender?

I am not really certain if my plugin really updates anything constantly in gui or sth but will probably need to investigate more

**Comment by chenpaner** [comment]
The current problem is to keep triggering ’depsgraph_update_handler_update_cutoff_distance‘, and no other problems have been found for the time being

**Comment by heisenshark** [comment]
I did not test anything with your plugin but I reduced some redraws of the ui in the dev branch, check it out, it might fix your issues

https://github.com/heisenshark/blender-krita-link-plugin/tree/develop

**Comment by chenpaner** [comment]
it fixed my issues,thanks!

**Comment by heisenshark** [comment]
fixed in [0.3](https://github.com/heisenshark/blender-krita-link-plugin/releases/tag/0.3)
